app/modules/ads/tracking/slot-tracker.js

```diff
@@ -6,10 +6,12 @@ const onRenderEndedStatusToTrack = [
   'success',
 ];
 const onChangeStatusToTrack = [
+  'success',
```

Comment:
It will cause multiple events in DW for single slot

Reply:
Ack, reverting.